packages/tldraw/src/Tldraw.tsx

```diff
@@ -216,6 +224,36 @@ export function Tldraw({
     setApp(newApp)
   }, [sId, id])
 
+  // In dev, we need to delete the prefixed
+  const entry =
+    window.location.port === '5420'
+      ? window.location.hash.replace('#/develop/', '')
```

Comment:
we have 2 apps in V1, there's one that uses the port 5420 and has the hash `#/develop/` that prevents us to get the search url and there's the other one 3000 (it's working with this one because we don't have the hash), that's why I checked if the port is 5420